### DISCUSSION TITLE
rocketmq 发送延时消息

### DISCUSSION THREAD

**Discussion by flylone** [apache/rocketmq#7022] [Q&A]
1.我的rocketmq服务端是5.x版本
2.我的rocketmq客户端版本是4.9.x版本(由于各种原因暂时不能升级到5.x)，我想用rocketmq的延时消息(延时时间级别那种不满足要求)，我在发送消息时设置TIMER_DELAY_SEC或TIMER_DELAY_MS，消息能在指定延时时间后消费

不知道这种方式合理的以及是否会有其它问题，

**Comment by RongtongJin** [comment]
> 1.我的rocketmq服务端是5.x版本 2.我的rocketmq客户端版本是4.9.x版本(由于各种原因暂时不能升级到5.x)，我想用rocketmq的延时消息(延时时间级别那种不满足要求)，我在发送消息时设置TIMER_DELAY_SEC或TIMER_DELAY_MS，消息能在指定延时时间后消费
> 
> 不知道这种方式合理的以及是否会有其它问题，

理论上可以的。可以测试一下

**Reply by flylone** [reply]
试了，是可以的
